packages/service-library/tests/aiohttp/long_running_tasks/conftest.py

```diff
@@ -0,0 +1,145 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+import asyncio
+import urllib.parse
+from typing import Awaitable, Callable
+
+import pytest
+from aiohttp import web
+from aiohttp.test_utils import TestClient
+from faker import Faker
+from pydantic import BaseModel, parse_obj_as
+from pytest_simcore.helpers.utils_assert import assert_status
+from servicelib.aiohttp import long_running_tasks
+from servicelib.aiohttp.long_running_tasks.server import (
+    TaskId,
+    create_task_name_from_request,
+)
+from servicelib.aiohttp.requests_validation import parse_request_query_parameters_as
+from servicelib.json_serialization import json_dumps
+from servicelib.long_running_tasks._task import TaskContext
+from tenacity._asyncio import AsyncRetrying
+from tenacity.retry import retry_if_exception_type
+from tenacity.stop import stop_after_delay
+from tenacity.wait import wait_fixed
+
+
+async def _string_list_task(
+    task_progress: long_running_tasks.server.TaskProgress,
+    num_strings: int,
+    sleep_time: float,
+    fail: bool,
+) -> web.Response:
+    generated_strings = []
+    for index in range(num_strings):
+        generated_strings.append(f"{index}")
+        await asyncio.sleep(sleep_time)
+        task_progress.update(message="generated item", percent=index / num_strings)
+        if fail:
+            raise RuntimeError("We were asked to fail!!")
+
+    # NOTE: this code is used just for the sake of not returning the default 200
+    return web.json_response(
+        data={"data": generated_strings}, status=web.HTTPCreated.status_code
+    )
+
+
+@pytest.fixture
+def task_context(faker: Faker) -> TaskContext:
+    return {"user_id": faker.pyint(), "product": faker.pystr()}
+
+
+@pytest.fixture
+def long_running_task_entrypoint() -> str:
+    return "long_running_entrypoint"
+
+
+@pytest.fixture
+def server_routes(
+    task_context: TaskContext, long_running_task_entrypoint: str
+) -> web.RouteTableDef:
+    routes = web.RouteTableDef()
+
+    class _LongTaskQueryParams(BaseModel):
+        num_strings: int
+        sleep_time: float
+        fail: bool = False
+
+    @routes.post("/long_running_task:start", name=long_running_task_entrypoint)
+    async def generate_list_strings(request: web.Request):
+        task_manager = long_running_tasks.server.get_tasks_manager(request.app)
+        query_params = parse_request_query_parameters_as(_LongTaskQueryParams, request)
+        assert task_manager, "task manager is not initiated!"
+        task_name = create_task_name_from_request(request)
```

Comment:
MINOR: again, naming ... make sure we identify these as long running tasks and not ``asyncio.tasks`` etc

Reply:
see other comment

Reply:
this is a test. also in this context there is no asyncio involved so I'd say that is ok.